component/cleanup-cronjob.libsonnet

```diff
@@ -0,0 +1,71 @@
+/*
+* Deploys the control-api billing entity cleanup cronjob
+*/
+local common = import 'common.libsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.control_api;
+local com = import 'lib/commodore.libjsonnet';
+
+local namespace = {
+  metadata+: {
+    namespace: params.namespace,
+  },
+};
+
+local hasCountriesConfig = params.odoo8.countries != null && std.length(params.odoo8.countries) > 0;
+
+local extraDeploymentArgs =
+  (if hasCountriesConfig then
+     [
+       '--billing-entity-odoo8-country-list=/config/billing_entity_odoo8_country_list.yaml',
+     ]
+   else
+     [])
+;
+
+
+local cronJob = kube.CronJob('cleanup-inflight-partner-records') + namespace {
+  spec+: {
+    schedule: params.cleanupJob.schedule,
+    failedJobsHistoryLimit: params.cleanupJob.jobHistoryLimit.failed,
+    successfulJobsHistoryLimit: params.cleanupJob.jobHistoryLimit.successful,
+    jobTemplate+: {
+      spec+: {
+        template+: {
+          spec+: {
+            nodeSelector: params.cleanupJob.nodeSelector,
+            restartPolicy: 'Never',
+            containers_+: {
+              cleanup: kube.Container('cleanup') {
+                image: '%(registry)s/%(image)s:%(tag)s' % params.images['control-api'],
+                args: common.MergeArgs(extraDeploymentArgs, params.cleanupJob.extraArgs),
+                env+: com.envList(params.cleanupJob.extraEnv),
+                volumeMounts+:
```

Comment:
I didn't notice yesterday but IIRC there's a `volumeMounts_` which helps you a bit (similar to `volumes_`). I don't remember off the top of my head how it works, so I can't make a suggestion. If you want you look it up you can adjust the code, but I don't mind either way.